src/frontend/src/components/Button/Button.styles.ts

```diff
@@ -0,0 +1,30 @@
+import { tv } from "tailwind-variants";
+
+const buttonStyles = tv({
+    base: [
+        // Base styles
+        "inline-flex items-center justify-center whitespace-nowrap rounded-[var(--spacing-4)] gap-2.5 transition-colors",
+        "font-bold text-button leading-normal",
+        "cursor-pointer",
+        // Focus, disabled, and pressed states from react-aria-components plugin
+        "focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-muted-blue-400 focus-visible:ring-offset-2",
+        "disabled:opacity-50 disabled:cursor-not-allowed",
+        "pressed:scale-[0.98]",
+    ],
+    variants: {
+        variant: {
+            primary: "bg-teal-800 text-white hover:bg-teal-950",
+            outline: "border border-gray-300 bg-transparent hover:bg-gray-100",
+            orange: "bg-orange-200 text-orange-800 hover:bg-orange-300",
```

Comment:
Can we also add the [secondary style variant](https://www.figma.com/design/LAy4vzpOLf7YnXbB7pGfN8/BHA-ECHO?node-id=14-41&m=dev)?

Reply:
I was wondering about that too! I saw the secondary button in the button component list, but did not get any luck finding where it is actually used in the mobile or desktop design. So I ended up removing it. I could add `secondary` back though.

Reply:
Thanks! Even if not immediately used, because we're implementing a headless library it might be helpful to implement every variant from the design system page so that if we need to add any other small UI down the road we've already got some form of cohesive styling setup